.github/workflows/generate-release-candidate.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate Release Candidate
+
+on: 
+  repository_dispatch:
+    types: [generate-release-candidate]
```

Comment:
Considering changing the dispatch name and the yml name to `staging-build-release-candidates` or similar.

Reply:
Changed to staging-build-release-candidate to be inconsistent with the other staging workflows.